charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -44,14 +44,14 @@ spec:
         - name: OPERATOR_NAME
           value: {{ template "zookeeper-operator.fullname" . }}
         {{- if .Values.additionalEnv }}
-{{ toYaml .Values.additionalEnv | indent 8 }}
+{{ toYaml .Values.additionalEnv | nindent 8 }}
         {{- end }}
         {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         {{- end }}
       {{- if .Values.additionalSidecars }}
-{{ toYaml .Values.additionalSidecars | indent 6 }}
+{{toYaml .Values.additionalSidecars | nindent 6 }}
```

Comment:
Why removing the space between {{ and toYaml?
You are not consistent with same call above

Reply:
This PR closed